README.md

```diff
@@ -43,11 +43,9 @@ Starting from version `0.22.0` of the library your project is required to use:
 ## Features
 
 * Code it like a boxed value `atomic(0)`, but run it in production efficiently:
-  * as `java.util.concurrent.atomic.AtomicXxxFieldUpdater` on Kotlin/JVM 
+  * as `java.util.concurrent.atomic.AtomicXxxFieldUpdater` on Kotlin/JVM
   * as a plain unboxed value on Kotlin/JS
-* Multiplatform: write common Kotlin code with atomics that compiles for Kotlin JVM, JS, and Native backends:
-    * Compile-only dependency for JVM and JS (no runtime dependencies)
-    * Compile and runtime dependency for Kotlin/Native 
+* Complete multiplatform support: JVM, Native, JS and Wasm (since Kotlin 1.9.20).
```

Comment:
@mvicsokolova does it make sense to clarify how different transformation flags  (from the [Transformation modes](https://github.com/Kotlin/kotlinx-atomicfu/blob/e10743be50e1caeb592679fab0618061820a115c/README.md#transformation-modes) and the [Options for post-compilation transformation](https://github.com/Kotlin/kotlinx-atomicfu/pull/334/files#options-for-post-compilation-transformation)) should work with Wasm?

Reply:
Yes, I think it's good to drop a word that the plugin does not affect Wasm target at all, and no transformations are applied.